lib/course.rb

```diff
@@ -0,0 +1,23 @@
+class Course
+  attr_reader :name,
+              :capacity,
+              :students
+
+  def initialize(name, capacity)
+    @name = name
+    @capacity = capacity
+    @students = []
+  end
+
+  def enroll(pupil)
+    @students << pupil
+  end
+
+  def full?
+    if @students.length < 2
```

Comment:
Watch out for hardcoding here! What can you replace 2 with? What attribute is available that makes more sense here?